storybookjs/storybook
# Angular: Fix flaky tests based on timezone

Issue: N/A

## What I did

When running the tests locally in GMT-7, I get a failing snapshot test against the angular snapshots. It seems to be due to their `| date` helper applying your timezone. I found that you can actually specify the format and timezone, and making those changes allows the tests to pass locally.

## How to test

- Is this testable with Jest or Chromatic screenshots? ✅ 
- Does this need a new example in the kitchen sink apps? ❌ 
- Does this need an update to the documentation? ❌ 

If your answer is yes to any of these, please make sure to include it in your PR.
